strategies/changed/src/main/java/org/arquillian/smart/testing/vcs/git/GitFetcher.java

```diff
@@ -12,19 +18,18 @@
 import org.eclipse.jgit.storage.file.FileRepositoryBuilder;
 import org.eclipse.jgit.treewalk.CanonicalTreeParser;
 
-class GitDiffFetcher {
+class GitFetcher {
```

Comment:
I think we can have a better name here. I didn't like `GitDiffFetcher`, but `GitFetcher` is potentially more confusing, as there is `git fetch` operation :) This component now does not only deals with diffs, but all changes, so I would propose `GitChangeResolver`?